### ISSUE TITLE
White Label - Document word]: In high contrast Aquatic mode, luminosity contrast ratio of keyboard focus indicator of controls present under video is less than 3:1

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5697]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

In High Contrast Aquatic mode, a low-contrast keyboard focus indicator makes it difficult for users with low vision or keyboard-only users to see which video control is active, leading to confusion, slower interaction, and potential errors, and creating an exclusionary experience.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Forced colors (high contrast mode)

### How to reproduce the issue?

Prerequisites:
Settings>Accessibility>Contrast themes>Aquatic/theme>Refresh/Reopen the application.

Repro Steps:
1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3.Navigate to white label experience chat page and type Video vimeo word in type your message text box.
4.Navigate to the search results.
5.Navigate on the video controls.
6.Verify that in high contrast Aquatic mode, luminosity contrast ratio of keyboard focus indicator of controls present under video is less than 3:1

### What do you expect?

In High Contrast Aquatic mode, the keyboard focus indicator for video controls should be clearly visible with a minimum 3:1 contrast ratio, allowing users to easily identify the focused control and navigate the interface effectively.

### What actually happened?

In high contrast Aquatic mode, luminosity contrast ratio of keyboard focus indicator of controls present under video is less than 3:1

### Do you have any screenshots or recordings to repro the issue?

<img width="1801" height="640" alt="Image" src="https://github.com/user-attachments/assets/4f04da84-7bd8-4c1c-8362-91c734102cf6" />

### Adaptive Card JSON

```json

```
